### ISSUE TITLE
Error message has "base" prepended

### ISSUE THREAD

**Issue by JustinasDaugela** [recurly/recurly-client-ruby#211]
When creating a subscription that already exists, the error message of `"base You already have a subscription to this plan."` is returned.

**Comment by csmb** [comment]
Strange. When I try to reproduce locally, I get the following:

`errors: {"base"=>["You already have a subscription to this plan."]}`

Could you provide some code to reproduce? Also, if you'd like the ability to add the same subscription to an account multiple times, you can let us know and they can take care of that.

**Comment by JustinasDaugela** [comment]
I'm just trying to create a subscription and rescuing the exception:
```
begin
  Recurly::Subscription.create!(
    plan_code: code,
    account: {
      account_code: account.account_code,
      billing_info: { token_id: token_id }
    }
  )
rescue => e
  puts e.message
end
```

Using version 2.4.6. I do not want the subscription to run multiple times, just wanted to present the user with the error message and saw it has 'base' prepended.

**Comment by lmnleaf** [comment]
Closed with #267
